translator/src/main/scala/io/kaitai/struct/testtranslator/specgenerators/RustSG.scala

```diff
@@ -10,7 +10,8 @@ import _root_.io.kaitai.struct.translators.RustTranslator
 
 class RustSG(spec: TestSpec, provider: ClassTypeProvider) extends BaseGenerator(spec) {
   val className = RustCompiler.type2class(spec.id)
-  val translator = new RustTranslator(provider, RuntimeConfig())
+  val importList = new CppImportList
+  val translator = new RustTranslator(provider, importList, RuntimeConfig())
```

Comment:
[`CppImportList`](https://github.com/kaitai-io/kaitai_struct_compiler/blob/2333448059e4887701102177420cd36e7b322ee3/shared/src/main/scala/io/kaitai/struct/languages/components/CppImportList.scala) is not used by Rust. This is used exclusively by C++, because there is a syntax difference between imports from standard library and local imports. [`RustCompiler`](https://github.com/kaitai-io/kaitai_struct_compiler/blob/2333448059e4887701102177420cd36e7b322ee3/shared/src/main/scala/io/kaitai/struct/languages/RustCompiler.scala#L11) uses [`ImportList`](https://github.com/kaitai-io/kaitai_struct_compiler/blob/2333448059e4887701102177420cd36e7b322ee3/shared/src/main/scala/io/kaitai/struct/ImportList.scala) like any other language. _It's defined in trait [`SingleOutputFile`](https://github.com/kaitai-io/kaitai_struct_compiler/blob/2333448059e4887701102177420cd36e7b322ee3/shared/src/main/scala/io/kaitai/struct/languages/components/SingleOutputFile.scala#L25), which [`RustCompiler`](https://github.com/kaitai-io/kaitai_struct_compiler/blob/2333448059e4887701102177420cd36e7b322ee3/shared/src/main/scala/io/kaitai/struct/languages/RustCompiler.scala#L15) uses._